plugins/plugin-core-support/src/lib/cmds/history/history.ts

```diff
@@ -163,11 +162,11 @@ export default (commandTree: CommandRegistrar) => {
   // commandTree.listen('/history/purge', historyModel.wipe, { docs: 'Clear your command history' })
 
   /** re-execute from history */
-  const againCmd = op => ({ argv, execOptions, parsedOptions: options }) => {
+  const againCmd = () => ({ argv, execOptions }) => {
     const N = argv[1] || historyModel.getCursor() - 2 // use the last command, if the user entered only "!!"
     console.error(execOptions)
     return again(N, execOptions && execOptions.history)
   }
-  const cmd = commandTree.listen('/!!', againCmd('!!'), { usage: usage.again('!!'), noAuthOk: true })
```

Comment:
Thanks! Removing ```const cmd = ``` will be enough for this case, otherwise the the `!!` command won't be listened.

Reply:
The other files look good!

Reply:
If you look at the line under this comment you'll see the line was not removed, only changed. Moreover, we can't drop the declaration, because it's used in the next line (L171).

Reply:
oh I see! Thanks!